Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -238,9 +240,7 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
             }
 
             handle(clientError: error)
-
-            let cachedServer = try cachedServer(registeredWith: keyPair)
```

Comment:
This will need more cleanup, but not in this hotfix.

Reply:
Good find, the `try` in the `cachedServer` call does indeed mask the real issue.